packages/clay-shared/src/Portal.tsx

```diff
@@ -24,10 +24,26 @@ export const ClayPortal: React.FunctionComponent<
 		 */
 		subPortalRef?: React.RefObject<Element>;
 	}
-> = ({children, containerRef, subPortalRef}) => {
+> = ({children, containerRef, subPortalRef, ...otherProps}) => {
 	const parentPortalRef = React.useContext(ClayPortalContext);
+
+	const createElement = (nodeName: string, attributes: any) => {
```

Comment:
It would be a good idea to move this method out of our component just to avoid creating a new reference or function declaration every time a new rendering occurs in the component.